training/ilab-wrapper/ilab

```diff
@@ -144,4 +145,12 @@ PODMAN_COMMAND=("sudo" "--preserve-env=$PRESERVE_ENV" "podman" "run" "--rm" "-it
     "--env" "HF_TOKEN"
     "${IMAGE_NAME}")
 
+sudo podman image exists "$IMAGE_NAME"
+if [ "$?" != "0" ]; then
+   echo "Initializing ilab container..."
+   id=$(sudo podman create "$SOURCE_IMAGE")
+   sudo podman commit "$id" "$IMAGE_NAME"
```

Comment:
This is creating a new layer in `/var/lib/containers` that references the base layers, but those layers will go away across an upgrade when the base ilab image changes, right?

IOW won't this break across upgrades?

Reply:
@cgwalters As long as we are updating our tag (1.1 today -> 1.2 tomorrow) this should be ok, it will then just create a new container at that point. We do need to add code to cleanup old images though.